packages/webpack/lib/configs/develop.js

```diff
@@ -108,7 +106,7 @@ module.exports = function getConfig(config) {
     },
     plugins: [
       new HotModuleReplacementPlugin(),
-      new NamedModulesPlugin(),
+      new HashedModuleIdsPlugin(),
```

Comment:
Wouldn't it be better to keep the `NamedModulesPlugin` here for easier debugging? 
Or is there a need to switch them?